web/packages/teleport/src/Bots/Details/InstancesPanel.tsx

```diff
@@ -27,8 +27,6 @@ import Flex from 'design/Flex/Flex';
 import { SortAscending, SortDescending } from 'design/Icon';
 import { Indicator } from 'design/Indicator/Indicator';
 import Text from 'design/Text';
-import { H2 } from 'design/Text/Text';
-import { fontWeights } from 'design/theme/typography';
```

Comment:
I'm not sure if this was the exact cause of those unused imports, but: if you set up your editor to run ESLint with `--fix` on save, then it should automatically remove unused imports for you. Bartosz added an ESLint rule for that a while ago. https://github.com/gravitational/teleport/pull/54953